packages/Chem/src/rdkit-service/rdkit-service-worker-client.ts

```diff
@@ -22,8 +22,8 @@ export class RdKitServiceWorkerClient extends WorkerMessageBusClient {
   freeMoleculesStructures = async () =>
     this.call(WORKER_CALL.FREE_MOLECULES_STRUCTURES);
 
-  getFingerprints = async (fingerprintType: Fingerprint) =>
-    this.call(WORKER_CALL.GET_FINGERPRINTS, [fingerprintType]);
+  getFingerprints = async (fingerprintType: Fingerprint, dict?: string[]) =>
```

Comment:
Why is it called "dict" if it's not a dictionary?

Reply:
Frankly, I just used the same name as it was in initMoleculeStructures(). Can't tell why it was named that way. But agree, it should be renamed. Will do.